services/src/main/java/org/keycloak/services/resources/WelcomeResource.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.keycloak.services.resources;
 
+import jakarta.ws.rs.core.*;
```

Comment:
Please revert this change to use `*` imports. I know we don't have a proper style guide, but this is one of the things that is good to avoid.

Reply:
IntelliJ did this for me, so perhaps we should add some configs to the repo to disable that default behaviour. Can you tell me why this is something we try to avoid?

Reply:
Fixed in 5fac8668ca5ccec016830d5c6ff65b072c8cb117